### PR TITLE
Overview page for micro-ROS demos

### PR DESCRIPTION
**Pull request by ralph-lange**

### CODE REVIEW

**Comment by ralph-lange** [comment]
@pablogs9, @FranFin, could you please take a photo of the ToF sensor board on white background and add it to this PR?

**Comment by FranFin** [comment]
> @pablogs9, @FranFin, could you please take a photo of the ToF sensor board on white background and add it to this PR?

**Comment by ralph-lange** [comment]
Just Added @FranFin's photo of the ToF sensor. PR is ready to be reviewed now.

File: _docs/tutorials/demos/overview/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Choose yourself:
+
+|              | [**Kobuki Demo**](../kobuki_demo/) | [**Crazyflie Demo**](../crazyflie_demo/) | [**ToF Sensor Demo**](../tof_demo/) |
+| --- | :-: | :-: | :-: |
+| _RTOS_       | NuttX                             | FreeRTOS                                | Zephyr                             |
+| _Hardware\*_ | Kobuki, Olimex LTD STM32-E407     | Crazyflie, radio, flow deck             | ST B-L475E-IOT01A Discovery kit    |
+| _HW Costs\*_ | ≈ 360$                            | ≈ 330$                                  | ≈ 60$                              |
+|              | <img src="kobuki.png" style="margin:auto;"/> | <img src="crazyflie.png" style="margin:auto;"/> |  <img src="tof_sensor.png" style="margin:auto;"/> |
+
+<div style="font-size:80%;color:gray;text-align:right;margin-bottom:1em;">*As a matter of course, you'll need a computer/laptop, various cables, and<br/>further auxiliary equipment. The hardware costs are only a rough estimate.</div>
```
Comment (FranFin) [inline _docs/tutorials/demos/overview/index.md:19]: Why this line break here?
'and<br/>further'

Comment (ralph-lange) [reply _docs/tutorials/demos/overview/index.md:19]: Just to put the text less prominently on the right-hand side.

**Pull request merged**